src/main/java/uk/gov/companieshouse/limitedpartnershipsapi/controller/IncorporationController.java

```diff
@@ -97,6 +100,35 @@ public ResponseEntity<Object> getIncorporation(
         }
     }
 
+    @GetMapping("/{" + URL_PARAM_INCORPORATION_ID + "}/validation-status")
+    public ResponseEntity<ValidationStatusResponse> getValidationStatus(
+            @RequestAttribute(TRANSACTION_KEY) Transaction transaction,
+            @PathVariable(URL_PARAM_INCORPORATION_ID) String incorporationId,
+            @RequestHeader(value = ERIC_REQUEST_ID_KEY) String requestId) throws ServiceException {
+        var logMap = new HashMap<String, Object>();
+        logMap.put(URL_PARAM_TRANSACTION_ID, transaction.getId());
+
+        try {
+            ApiLogger.infoContext(requestId, "Calling service to validate a Limited Partnership Incorporation", logMap);
+            var validationStatus = new ValidationStatusResponse();
```

Comment:
Just wondering why we don't call the service and check that errors are empty first and then if empty set to true, if not false.